products/workflows/api.yaml

```diff
@@ -0,0 +1,95 @@
+    name: beta
+    base_url: https://workflows.googleapis.com/v1beta/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Workflows API
+    url: https://console.cloud.google.com/apis/library/workflows.googleapis.com/
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: True
+    allowed:
+      - True
+      - False
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Workflow'
+    min_version: beta
+    description: |
+      Workflow program to be executed by Workflows.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Managing Workflows':
+          'https://cloud.google.com/workflows/docs/creating-updating-workflow'
+      api: 'https://cloud.google.com/workflows/docs/reference/rest/v1beta/projects.locations.workflows'
+    base_url: projects/{{project}}/locations/{{region}}/workflows
+    create_url: projects/{{project}}/locations/{{region}}/workflows?workflowId={{name}}
+    update_verb: :PATCH
+    update_mask: true
+    # update_url: projects/{{project}}/topics/{{name}}
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: region
+        description: The region of the workflow.
+        url_param_only: true
+        input: true
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        required: true
+        description: Name of the Workflow.
+        input: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: |
+          Description of the workflow provided by the user. Must be at most 1000 unicode characters long.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: |
+          A set of key/value label pairs to assign to this Workflow.
+      - !ruby/object:Api::Type::String
+        name: 'serviceAccount'
+        input: true
+        pattern: 'projects/*/serviceAccounts/{{service_account}}'
+        description: |
+          Name of the service account associated with the latest workflow version. This service account represents the identity of the workflow and determines what permissions the workflow has.
+      - !ruby/object:Api::Type::String
+        name: 'sourceContents'
+        input: true
+        description: |
+          Workflow code to be executed. The size limit is 32KB.
+      - !ruby/object:Api::Type::String
+        name: 'revisionId'
+        output: true
+        description: |
+          The revision of the workflow. A new one is generated if the service account or source contents is changed.
```

Comment:
Missing output fields `state`, `createTime`, `updatetime`, `revisionCreateTime`?

Reply:
revisionCreateTime is redundant in our case since we will delete and recreate the entire workflow. I added the other ones.